packages/integration-tests/cmr.js

```diff
@@ -328,6 +299,7 @@ async function getOnlineResourcesUMMG(cmrLink) {
  *
  * @returns {Promise<Array<Object>>} - Promise returning array of links
  */
+// eslint-disable-next-line no-unused-vars
 async function getOnlineResources({ cmrMetadataFormat, cmrConceptId, cmrLink }) {
```

Comment:
Same deal, just a brief documentation about what's happening might be helpful in six months when we've forgotten all of this.